test/com/linkedin/drelephant/configurations/fetcher/FetcherConfigurationTest.java

```diff
@@ -105,5 +116,18 @@ public void testParseFetcherConf4() {
         + " classname com.linkedin.drelephant.mapreduce.MapReduceFetcherHadoop2");
     FetcherConfiguration fetcherConf = new FetcherConfiguration(document4.getDocumentElement());
   }
+
+  /**
+   *  Test Spark fetcher params, Event log size and log directory
+   */
+  @Test
+  public void testParseFetcherConf5() {
```

Comment:
Should either add a new test or update an existing test to check the right log dir is set when nothing is set in the config.